src/main/java/edu/utdallas/davisbase/representation/InsertCommandRepresentation.java

```diff
@@ -0,0 +1,61 @@
+package edu.utdallas.davisbase.representation;
+
+import java.util.List;
+
+public class InsertCommandRepresentation implements CommandRepresentation {
+  private String command;
+  private String table;
+  private List<String> columns;
+  private List<String> values;
+
+  public InsertCommandRepresentation(String command, String table, List<String> columns, List<String> values) {
+    this.command= command;
+    this.table = table;
+    this.columns = columns;
```

Comment:
```suggestion
    this.columns = Collections.unmodifiableList(new ArrayList<>(columns));
```

(compile-time defensive programming + run-time fail-fast immutability)